Cartfile

```diff
@@ -1 +1 @@
-github "ReactiveCocoa/ReactiveCocoa" "v3.0-beta.1"
+github "ReactiveCocoa/ReactiveCocoa" ~> 3.0
```

Comment:
I had that originally but switched it because this plays nicer with transitive dependencies (albeit by accident). I'll go ahead and target the most recent beta directly.